src/app.html

```diff
@@ -4,9 +4,9 @@
         <meta charset="utf-8" />
         <meta
             name="description"
-            content="Appwrite is an open-source platform for building applications at any scale, using your preferred programming languages and tools." />
-        <link rel="icon" type="image/svg+xml" href="%sveltekit.assets%/logos/appwrite-icon.svg" />
-        <link rel="mask-icon" type="image/png" href="%sveltekit.assets%/logos/appwrite-icon.png" />
+            content="Build something real. Turn your ideas into real products by chatting with the most complete AI builder available to date" />
+        <link rel="icon" type="image/svg+xml" href="%sveltekit.assets%/logos/imagine-icon.svg" />
+        <link rel="mask-icon" type="image/png" href="%sveltekit.assets%/logos/imagine-icon.png" />
```

Comment:
this changes it also for appwrite, we need to do it based on the resolved profile 👍🏻 

there is a `%sveltekit.env.PUBLIC_CONSOLE_PROFILE%` placeholder we can use